AudioTranscriber.py

```diff
@@ -1,19 +1,20 @@
 import os
+import queue
 import threading
 import io
 from datetime import timedelta
 import wave
 import tempfile
-import whisper
 import custom_speech_recognition as sr
 import pyaudiowpatch as pyaudio
 from heapq import merge
+import conversation
 
 PHRASE_TIMEOUT = 3.05
 
 
 class AudioTranscriber:
-    def __init__(self, mic_source, speaker_source, model):
+    def __init__(self, mic_source, speaker_source, model, convo: conversation.Conversation):
```

Comment:
What are the types of other arguments. Should add them as well.

Reply:
Good point. I am adding as I go along. Will add more in the next PR.